service/src/test/resources/application-test.yaml

```diff
@@ -0,0 +1,160 @@
+deploy_env: ${DEPLOY_ENV:dev} # since there's a default, define this once
```

Comment:
This is a test config file that adds some providers to test specific cases that don't yet appear in production.